spec.md

```diff
@@ -513,6 +521,29 @@ function fooBarBaz($arg1, &$arg2, $arg3 = [])
 }
 ```
 
+If a function or method contains no statements (such as a no-op implementation or when using
+constructor property promotion), then the body SHOULD be abbreviated as `{}` and placed on the same
+line as the previous symbol, separated by a space.  For example:
+
+```php
+class Point
+{
+    public function __construct(private int $x, private int $y) {}
+    
+    // ...
+}
+```
+
+```php
+class Point
+{
+    public function __construct(
+      public readonly int $x,
+      public readonly int $y,
+    ) {}
```

Comment:
That is inconsistent with regular method declaration, I don't think it worth making this case special.